tests/test_io.py

```diff
@@ -9,6 +9,14 @@
 
 from .conftest import TEST_DATA_DIR, META_DF
 
+try:
```

Comment:
Perhaps this could be moved into `__init__.py`so that `has_xlrd` becomes a global variable.
This could be used to raise a pyam exception if a user attempts to read or write an "xls" file instead of relying on pandas to do so. The advantage might be that we could directly point the user to "please install with the optional_io_formats option or use 'xslx'".

Reply:
I would rather keep it here - catching a missing `xlrd` package in pyam again introduces the risk that pyam and pandas diverge in their behavior. Plus the pandas-error-message is sufficiently clear and probably easier to understand than pointing a user to "optional_io_formats"...